### PR TITLE
i18n(es): Updated `components.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `components.mdx` based on this commit 73eb5e6ac6511dc4a6f5c4ca6c0c60d521f1db3c

### CODE REVIEW

File: docs/src/content/docs/es/guides/components.mdx [hunk 1]
```diff
@@ -140,6 +140,50 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
...
+<CardGrid>
+	<LinkCard
+		title="Creación de contenido en Markdown"
+		href="/guides/authoring-content/"
+	/>
+	<LinkCard title="Componentes" href="/guides/components/" />
+</CardGrid>
+```
+
+El código de arriba genera lo siguiente en la página:
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:169]: We use this phrasing elsewhere on the page, so maybe we should here too?

```suggestion
El código anterior genera lo siguiente en la página:
```

File: docs/src/content/docs/es/guides/components.mdx [hunk 2]
```diff
@@ -140,6 +140,50 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
...
+	href="/guides/customization/"
+/>
+
+<CardGrid>
+	<LinkCard
+		title="Creación de contenido en Markdown"
+		href="/guides/authoring-content/"
+	/>
+	<LinkCard title="Componentes" href="/guides/components/" />
+</CardGrid>
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:166]: I think we can use the Spanish URLs here:
```suggestion
<LinkCard
	title="Personalizando Starlight"
	description="Aprende a hacer que tu sitio Starlight sea único con estilos personalizados, fuentes y más."
	href="/es/guides/customization/"
/>

<CardGrid>
	<LinkCard
		title="Creación de contenido en Markdown"
		href="/es/guides/authoring-content/"
	/>
	<LinkCard title="Componentes" href="/es/guides/components/" />
</CardGrid>
```

File: docs/src/content/docs/es/guides/components.mdx [hunk 3]
```diff
@@ -140,6 +140,50 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
...
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:185]: And here too:

```suggestion
<LinkCard
	title="Personalizando Starlight"
	description="Aprende a hacer que tu sitio Starlight sea único con estilos personalizados, fuentes y más."
	href="/es/guides/customization/"
/>

<CardGrid>
	<LinkCard
		title="Creación de contenido en Markdown"
		href="/es/guides/authoring-content/"
	/>
	<LinkCard title="Componentes" href="/es/guides/components/" />
</CardGrid>
```

**Review by delucis — COMMENTED** [review]
Thanks for the speedy translation @dreyfus92! Left a few comments.

**Comment by dreyfus92** [comment]
Thanks for the review @delucis 😁

File: docs/src/content/docs/es/guides/components.mdx [hunk 4]
```diff
@@ -140,6 +140,50 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
...
+
+Agrupa varios componentes `<LinkCard>` en `<CardGrid>` para mostrar las tarjetas una al lado de la otra cuando hay suficiente espacio.
+
+```mdx
+import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+
+<LinkCard
+	title="Personalizando Starlight"
+	description="Aprende a hacer que tu sitio Starlight sea único con estilos personalizados, fuentes y más."
+	href="/guides/customization/"
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:157]:
```suggestion
	href="/es/guides/customization/"
```

File: docs/src/content/docs/es/guides/components.mdx [hunk 5]
```diff
@@ -140,6 +140,50 @@ Utiliza una cuadrícula de tarjetas en tu página de inicio para mostrar las car
...
+```
+
+El código anterior genera lo siguiente en la página:
+
+import { LinkCard } from '@astrojs/starlight/components';
+
+<LinkCard
+	title="Personalizando Starlight"
+	description="Aprende a hacer que tu sitio Starlight sea único con estilos personalizados, fuentes y más."
+	href="/guides/customization/"
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:176]:
```suggestion
	href="/es/guides/customization/"
```

**Review by delucis — COMMENTED** [review]
Oops. Looks like a couple of the link suggestions went missing. Here they are again.

**Comment by dreyfus92** [comment]
woops, my bad 😅

**Review by delucis — APPROVED** [review]
Let’s go! 😃

**Pull request merged**